packages/test/src/index.ts

```diff
@@ -0,0 +1,250 @@
+import {
+    AggregateEpochKeysProof,
+    AGGREGATE_KEY_COUNT,
+    Circuit,
+    defaultEpochTreeLeaf,
+    EPOCH_TREE_ARITY,
+    EPOCH_TREE_DEPTH,
+    SignupProof,
+    STATE_TREE_DEPTH,
+} from '@unirep/circuits'
+import { defaultProver } from '@unirep/circuits/provers/defaultProver'
+import {
+    genRandomSalt,
+    hash4,
+    IncrementalMerkleTree,
+    SparseMerkleTree,
+    stringifyBigInts,
+    ZkIdentity,
+} from '@unirep/utils'
+
+// users
+export async function bootstrapUsers(attester, epoch, unirepContract) {
+    const stateTree = new IncrementalMerkleTree(STATE_TREE_DEPTH)
+    const randomUserNum = Math.ceil(Math.random() * 5)
```

Comment:
What do you think of making this value much larger? Maybe like 100 or 1000?

Reply:
yeah I think we can make the number of users (attestations) to be an input of the function
so we can test the number we want